README.md

```diff
@@ -1 +1,85 @@
-# teams-api
\ No newline at end of file
+# @nozbe/teams-api
+
+<h4 align="center">
+  A npm package to connect with Nozbe Teams ✅
+</h4>
+
+<p align="center">
+  <a href="https://github.com/Nozbe/teams-api/blob/master/LICENSE">
+    <img src="https://img.shields.io/badge/License-MIT-blue.svg" alt="MIT License">
+  </a>
+
+  <!-- <a href="https://travis-ci.com/Nozbe/teams-api">
+    <img src="https://api.travis-ci.com/Nozbe/teams-api.svg?branch=master" alt="CI Status">
+  </a> -->
+
+  <!-- <a href="https://www.npmjs.com/package/@nozbe/watermelondb">
+    <img src="https://img.shields.io/npm/v/@nozbe/watermelondb.svg" alt="npm">
+  </a> -->
+</p>
+
+<!-- |     | @nozbe/teams-api                                               |
+| --- | -------------------------------------------------------------- |
+| ⚡️ | **Launch your app instantly** no matter how much data you have | -->
+
+## What is `@nozbe/teams-api`?
+
+**@nozbe/teams-api** is a way to interact with your [Nozbe Teams](https://teams.nozbe.com) account.
+
+It allows you to create utilities for Nozbe Teams, list your projects, manage your tasks and comment on them. For now - It is continuosly developed to give you more possibilities.
+
+> ⚠️ Note this package is still a WIP and the API may change (even if we do what we can to avoid that).
+
+## Installation
+
+```
+npm install @nozbe/teams-api
+```
+
+or
+
+```
+yarn add @nozbe/teams-api
+```
+
+## Usage
+
+**Quick (over-simplified) example:** adding the task to your Single Actions
```

Comment:
```suggestion
**Quick (over-simplified) example:** adding the task to your Single tasks
```

`single actions` is an old name, though the REST API doesn't reflect that (because backwards compatibility)